src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ Licensed to the Apache Software Foundation (ASF) under one or more
+~ contributor license agreements. See the NOTICE file distributed with
+~ this work for additional information regarding copyright ownership.
+~ The ASF licenses this file to You under the Apache license, Version 2.0
+~ (the "License"); you may not use this file except in compliance with
+~ the License. You may obtain a copy of the License at
+~
+~      http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software
+~ distributed under the License is distributed on an "AS IS" BASIS,
+~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+~ See the license for the specific language governing permissions and
+~ limitations under the license.
+-->
+<Configuration status="ERROR">
+  <properties>
+    <property name="LOG_DIR">${env:JENKINS_HOME}/logs</property>
+  </properties>
+  <Appenders>
+
+    </RollingFile>
+    <RollingFile name="audit" fileName="${LOG_DIR}/audit.log" filePattern="${LOG_DIR}/archive/audit.log.%d{yyyyMMdd_HHmmss}-%i">
+      <RFC5424Layout enterpriseNumber="38143" includeMDC="true" mdcId="RequestContext" appName="Log4j-audit-plugin" mdcPrefix="" newLine="true"/>
```

Comment:
I wonder if we should make the enterprise number customizable? Or should this number reflect the application itself?

Reply:
as per discussion new ticket created for it :)
[JENKINS-54867](https://issues.jenkins-ci.org/browse/JENKINS-54867)